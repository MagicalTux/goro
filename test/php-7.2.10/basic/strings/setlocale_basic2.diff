     *** Testing setlocale() with an array containing list of locales ***
002- -- Testing setlocale() : 'category' argument as LC_ALL & 'locale' argument as an array --
003- string(%d) "%s"
004- Checking currency settings in the new locale, expected: %s, Found: %s
005- Test PASSED.
006- Done
