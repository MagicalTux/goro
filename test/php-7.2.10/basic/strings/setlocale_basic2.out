*** Testing setlocale() with an array containing list of locales ***